test/libsolidity/smtCheckerTests/types/address_staticcall.sol

```diff
@@ -10,14 +10,16 @@ contract C
 		assert(success);
 		assert(x == 0);
 		assert(map[0] == 0);
-		assert(localMap[0] == 0);
+		// Disabled because of Spacer's seg fault
+		//assert(localMap[0] == 0);
```

Comment:
Is that on macOS again or on Linux too?

Reply:
Also on Linux, the current state of this PR is what I got locally.

Reply:
Is that maybe this? https://github.com/Z3Prover/z3/issues/4690

Reply:
Yea I guess